engine/execution/computation/manager.go

```diff
@@ -232,7 +235,7 @@ func DefaultFVMOptions(chainID flow.ChainID, extensiveTracing bool, scheduleCall
 				runtime.Config{},
 			),
 		),
-		fvm.WithEVMEnabled(true),
+		fvm.WithEVMEnabled(evmEnabled),
```

Comment:
Maybe we should also ensure this isn't just the default, but actually override the context option when we read/use it from the context?